image/test_script.py

```diff
@@ -0,0 +1,21 @@
+#Test Script
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.